src/main/java/cypherchatbot/util/CommandReader.java

```diff
@@ -0,0 +1,101 @@
+package cypherchatbot.util;
+
+import cypherchatbot.CypherException;
+import cypherchatbot.command.ByeCommand;
+import cypherchatbot.command.Command;
+import cypherchatbot.command.DeadlineCommand;
+import cypherchatbot.command.DeleteCommand;
+import cypherchatbot.command.EventCommand;
+import cypherchatbot.command.FindCommand;
+import cypherchatbot.command.HelpCommand;
+import cypherchatbot.command.ListCommand;
+import cypherchatbot.command.MarkCommand;
+import cypherchatbot.command.ToDoCommand;
+import cypherchatbot.command.UnmarkCommand;
+
+public class CommandReader {
+
+    private static enum Commands {
+
+        LIST, TODO, EVENT, DEADLINE, MARK, UNMARK, BYE, HELP, DELETE, FIND
+
+    }
+
+    /**
+     * Parses through the given input by user and instantiates a new Command Class through the use
+     * of enums and switch case statements.
+     *
+     *
+     * @param input The string input given by the user
+     * @return A Command Object depending on the input given by the user
+     * @throws CypherException Throws a custom exception if an invalid input is given by the user
```

Comment:
I think it is not necessary to mention "Throws a custom exception" as it is implied. 

```suggestion
     * @throws CypherException If an invalid input is given by the user.
```

I believe this is more closely aligned with the Javadoc conventions as well.